docs/content/preview/releases/techadvisories/ta-20827.md

```diff
@@ -0,0 +1,40 @@
+---
+title: TA-20827
+headerTitle: Correctness issue for queries using SELECT DISTINCT
+headcontent: 02 Feb 2024
+type: docs
+showRightNav: true
+cascade:
+  unversioned: true
+rightNav:
+  hideH2: true
+type: docs
+---
+
+|          Product           |  Affected Versions  |  Related Issues   | Fixed In |
+| :------------------------- | :------------------ | :---------------- | :------- |
+| {{<product "ysql">}}       | {{<release "2.20.1.x">}} | {{<issue 20827>}} | {{<release "2.20.2.0, 2.21.1.0">}}      |
```

Comment:
same issue about the link for affected versions - Its ideally supposed to link to 2.20.1.0. But as it stops at 2.20.1.. it just lands on the page.